### ISSUE TITLE
rook-ceph deployments fails for non-HA clusters

### ISSUE THREAD

**Issue by laserK3000** [argoflow/argoflow#55]
I encountered the problem of rook-ceph failing to deploy on non-HA clusters. As the reason is not immediately apparent, I suggest adding a single node rook-ceph config for testing environments.

**Comment by laserK3000** [comment]
https://github.com/argoflow/argoflow/pull/56

**Comment by DavidSpek** [comment]
Ceph really isn't meant for single device deployments. May I ask where you are running your cluster? It might make more sense to use local volumes if you are using just a single node on a workstation, or to have Ceph create 3 OSDs for your single drive (assuming it is an SSD) this would also increase performance.

**Comment by laserK3000** [comment]
I agree that Ceph is not meant for single node deployments. I run an on-prem setup plus a second single node machine for testing. The idea behind adding it was that one has at least something to test rook-ceph on.
Nevertheless it would be helpfull to add that one has to do some additional configuration the get Argoflow running on a non-HA setup. The Readme somehow suggests that for a non-HA setup removing `/ha` from the URI in `./argocd/kustomization.yaml` is sufficient.
